pkg/host/kernel_test.go

```diff
@@ -0,0 +1,199 @@
+package host
+
+import (
+	"os"
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/k8snetworkplumbingwg/sriov-network-operator/pkg/consts"
+	"github.com/k8snetworkplumbingwg/sriov-network-operator/pkg/vars"
+	"github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/fakefilesystem"
+)
+
+const (
+	testUnknownDev            = "unknown-dev"
+	testUnknownDriver         = "unknown-driver"
+	testDev                   = "0000:d8:00.0"
+	testDevPath               = "/sys/bus/pci/devices/" + testDev
+	testDevDriverPath         = testDevPath + "/driver"
+	testDevDriverOverridePath = testDevPath + "/driver_override"
+	testDriversRelPath        = "../../../../bus/pci/drivers/"
+	testDevDriverSymlink      = testDriversRelPath + testDriver
+	testDevDriver2Symlink     = testDriversRelPath + testDriver2
+	testDriver                = "test-driver"
+	testDriverPath            = "/sys/bus/pci/drivers/" + testDriver
+	testDriverBindPath        = testDriverPath + "/bind"
+	testDriverUnbindPath      = testDriverPath + "/unbind"
+	testDriver2               = "vfio-pci" // dpdk
+	testDriver2Path           = "/sys/bus/pci/drivers/" + testDriver2
+	testDriver2BindPath       = testDriver2Path + "/bind"
+	testDriver2UnbindPath     = testDriver2Path + "/unbind"
+	testPciDeviceProbePath    = "/sys/bus/pci/drivers_probe"
+)
+
+func testFileContent(path, expectedContent string) {
```

Comment:
please, rename to `assertFileContentsEquals(path, expectedContent string)`

Reply:
done